src/main/java/homework/home10012019/Fruit.java

```diff
@@ -33,15 +39,19 @@ public int getDayToLive() {
 		return dayToLive;
 	}
 
-	public void setDayToLive(int dayToLive) {
+//TODO checked exception in this method:
```

Comment:
Dont set "TODO" in comments

Reply:
Todo is spesial point - for understand whay you didn't write now, but must will write in future

Reply:
ok. I understand, i will no use TODO :)